Moonscraper Chart Editor/Assets/Scripts/Game/NoteVisuals/NoteVisuals3DManager.cs

```diff
@@ -102,12 +104,12 @@ protected override void Awake ()
                 {
                     switch (visualNoteType)
                     {
-                        case Note.NoteType.Tap:
+                        case VisualNoteType.Tap:
                             {
                                 colorMat = resources.GetToolTapMaterial(gameMode, laneInfo, noteIndex);
                                 break;
                             }
-                        case Note.NoteType.Cymbal:
+                        case VisualNoteType.Cymbal:
```

Comment:
Handle case for Tom, let it fall through to default just to make things a bit clearer.